lib/active_model/serializer/adapter.rb

```diff
@@ -1,7 +1,10 @@
+require 'active_model/serializer/cache'
+
 module ActiveModel
   class Serializer
     class Adapter
       extend ActiveSupport::Autoload
+      include ActiveModel::Serializer::Cache
```

Comment:
@kurko, to avoid performance loss I'll just remove the fragment cache logic, that correspond for 90% of all code, agreed?:
```ruby
ef cached_object
         klass = serializer.class
        if klass._cache && !klass._cache_only && !klass._cache_except
          _cache_key = (klass._cache_key) ? "#{klass._cache_key}/#{serializer.object.id}-#{serializer.object.updated_at}" : serializer.object.cache_key
          klass._cache.fetch(_cache_key, klass._cache_options) do
            yield
          end

        elsif klass._cache_only && !klass._cache_except || !klass._cache_only && klass._cache_except
          FragmentCache.new(self, serializer).fetch
        else
          yield
        end
      end
```